docs/contracts.rst

```diff
@@ -367,9 +367,9 @@ inheritable properties of contracts and may be overridden by derived contracts.
         }
 
         /// This function is protected by a mutex, which means that
-        /// reentrant calls from within msg.sender.call cannot call f again.
-        /// The `return 7` statement assigns 7 to the return value but still
-        /// executes the statement `locked = false` in the modifier.
+        /// reentrant calls from within "msg.sender.call" cannot call "f" again.
```

Comment:
The backticks are used to format code, I think it is good to keep them unless you have a compelling reason.

Reply:
Which is the standard? There are two different styles in this file.

Reply:
Ah I'm sorry, I didn't see that. I would lean slightly towards ` `. What do you think, @axic?

Reply:
I applied this throughout this file with the latest commit.